### PR TITLE
Disable tests related to GitHub issue #204

### PR DESCRIPTION
**Pull request by jasonmalinowski**
Disable tests related to issue #204.

### CODE REVIEW

File: src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs [hunk 1]
```diff
@@ -125,7 +125,7 @@ private void SynchronousContinuationsDoNotRunWithinGetValueCallCore(TaskStatus e
             Assert.Equal(expectedTaskStatus, observedAntecedentTaskStatus.Value);
         }
 
-        [Fact]
+        [Fact(Skip = "GitHub Issue 204")]
```
Comment (tmeschter) [inline src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs]: Maybe put in the URL to the issue instead of "GitHub Issue 204"?

**Comment by jaredpar** [comment]
:+1:

File: src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs (hunk 1 above)
Comment (jasonmalinowski) [inline src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs]: Do we have a standard for this?

Comment (jaredpar) [inline src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs]: Lets make the standard to paste in the URL.  Makes it clickable.

Comment (srivatsn) [inline src/Workspaces/CoreTest/UtilityTest/AsyncLazyTests.cs]: +1

**Pull request merged**